scripts/st2bootstrap-deb.sh

```diff
@@ -113,6 +113,31 @@ setup_args() {
   fi
 }
 
+function get_package_url() {
+  # Retrieve direct package URL for the provided dev build, subtype and package name regex.
+  DEV_BUILD=$1 # Repo name and build number - <repo name>/<build_num> (e.g. st2/5646)
+  DISTRO=$2  # Distro name (e.g. trusty,xenial,el6,el7)
+  PACKAGE_NAME_REGEX=$3
+
+  PACKAGES_METADATA=$(curl -Ss -q https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts)
+
+  if [ -z "${PACKAGES_METADATA}" ]; then
+      echo "Failed to retrieve packages metadata from https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts"
+      exit 2
```

Comment:
We can use more clean `return` instead of `exit` for a function

Reply:
Sadly return won't make a difference with regards to `set +e`. If `-e` mode is enabled, global error handler will still be called in case the function exits with non zero or returns non zero.